IronGram/build/resources/main/static/main.js

```diff
@@ -1,22 +1,44 @@
-function getPhotos(photoData){
-    for(var i in photoData){
+function getPhotos(photoData) {
+    for (var i in photoData) {
         var photo = photoData[i];
+        var elemDelayTime = photo.lifeDuration;
         var elem = $("<img>");
         elem.attr("src", "photos/" + photo.filename);
         elem.attr("height", 200);
         elem.attr("width", 200);
 
         $("#photos").append(elem);
+        $(elem).fadeIn(300).delay(elemDelayTime).fadeOut(500);
+    }
+    $.post("/delete");
```

Comment:
This is an innovative solution.  As discussed in class the downside is that just turning off javascript on the client computer will stop your system from deleting the images when intended.  The back end system should generally be in charge of any critical processes as the client has no ability to impact the outcome intentionally or unintentionally.